lib/ansible/module_utils/netcli.py

```diff
@@ -47,6 +47,11 @@ def __init__(self, msg, failed_conditions):
         super(FailedConditionsError, self).__init__(msg)
         self.failed_conditions = failed_conditions
 
+class FailedConditionalError(Exception):
+    def __init__(self, msg, conditional):
```

Comment:
I would expect the arg name here to be 'failed_conditional'  based on the names in FailedConditionsError and the
'self.failed_conditional = conditional' line. But I don't really understand how it's used, so that may not make sense.